### ISSUE TITLE
docs: Running list of Eng / Design improvements for 2025

### ISSUE THREAD

**Issue by dgmccart** [chanzuckerberg/cryoet-data-portal#1350]
**Engineering**

- [ ] Custom examples of inherited functions #890 
- [ ] Autogeneration of data model
- [ ] Versioned docs

**Improving Design**

- [ ] Improved API docs styling
- [ ] Accordion / admonition styling
- [ ]

**Comment by melissawm** [comment]
Effort estimation:

Engineering

- Custom examples of inherited functions **(1h)**
- Autogeneration of data model **(on hold, waiting for eng suggestions)**
- Python scripts or notebooks generation for examples and tutorials **(10h)**
- Versioned docs **(10h)**

Improving Design

- Improved API docs styling
    - I believe this is about the "attributes in table" issue. **(10h)**
- Accordion / admonition styling **(5h)**
- Add ability to link and copy link to specific questions in the [doc site faq](https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_faq.html) ( addressing https://github.com/chanzuckerberg/cryoet-data-portal/issues/419) - this may be an engineering issue, waiting on confirmation
